README.md

```diff
@@ -156,6 +156,7 @@ Terraform documentation is generated automatically using [pre-commit hooks](http
 | this\_rds\_cluster\_arn | The ID of the cluster |
 | this\_rds\_cluster\_database\_name | Name for an automatically created database on cluster creation |
 | this\_rds\_cluster\_endpoint | The cluster endpoint |
+| this\_rds\_cluster\_engine_version | The computed engine version |
```

Comment:
```suggestion
| this\_rds\_cluster\_engine_version | The cluster engine version |
```

Reply:
Docs are generated from `description` in variables and outputs, so updating it here.